### PR TITLE
Explain attribute access of the @private decorator.

### LINKED ISSUE #62: Document @private attribute protection and method access.
* explain why we fight against procedural code with a lot of coupling, low cohesion, and shared data flowed from top to bottom.

### PR DESCRIPTION
**Pull request by proofit404**
Fix #62

### CODE REVIEW

**Comment by proofit404** [comment]
@supadrupa, please review :pray:

File: docs/private.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+>>> user.greet()
+'Hello, Jeff'
+
+>>> user._name  # No information hiding here :(
+'Jeff'
+
+```
+
+It maybe a nice convention to have in the code base, but
```
Comment (supadrupa) [inline docs/private.md:39]: `It may be a nice convention to have it in the code base, but`

File: docs/private.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+encapsulation and **information hiding**.
+
+Procedural programming have a lot of problems related to coupling.
+All data flows directly through the execution flow. Every part of
+the code in this flow is highly coupled with each other because of the
+data they pass to each other.
+
+In opposite in the object-oriented programming we put parts of the
+data close to the parts of the code where it'll be used. Thus the
+execution flow see only method calls. Client code does not know what
```
Comment (supadrupa) [inline docs/private.md:148]: `Thus the execution flow sees only method calls.`

File: docs/private.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+data were encapsulated inside the object. And it's non of its
```
Comment (supadrupa) [inline docs/private.md:149]: `And it's none of its business.`

File: docs/private.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+business.
+
+Thus all attributes encapsulated inside the objects are private. The
+constructor is the only place you can put anything inside the object.
```
Comment (supadrupa) [inline docs/private.md:153]: `The constructor is the only place where you can put anything inside the object.`

**Review by supadrupa — COMMENTED** [review]
Done

**Comment by proofit404** [comment]
@ditansu, please review 🙏

**Comment by proofit404** [comment]
@supadrupa thanks!

**Comment by hupptechnologies** [comment]
[Banner](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
franee, @Thavatchai-lab, @0xwilliampeter, @D3athGr1p, @vthinkxie, @raydzast, @yorennz, @Huge, @GuilhermeC18, @tony92151, @yuanjunliang, @bwash105, @ZiJiaW, @lucoo01, @vikusik-simpapusik, @claudiopb, @lutz, @ehelvacikoylu, @ikozigbo, @yrzx404, @HelloMokuzai, @cybernetics, @chauhansagun, @OftenUser, @muflihunaf, @hambalee, @westerncj, @Meharab, @0xCFx, @MrPops, @Mi-kio, @mengguang, @darshakthakore, @gaurav1911, @SiddharthaMadhav, @vinmay, @jerrywjh, @SharmaRajan, @DylanHe, @simon-xia, @Duy-Thanh, @valkx, @defconst, @Grrr1Want2, @juhi24, @chuyenbka, @fcjz, @pc163, @najahlino, @stenppfnDevious3901, @zzf03680147, @sjin09, @willshome, @halleluyhwhyeshua, @roshanmirajkar, @otaha178, @drpaulow, @RaoPrashanth, @ngoct47449, @ANTCPU, @mahfuzali, @nlipartiia-hacken, @yizhzhang, @yabeen, @baldychristophe, @maddox-j, @12knnights, @csawe, @asreejith-vcrs, @deniskv, @jebron, @gc87, @autoshgame, @rahife, @gsxgoldenlegendary, @sun-driedtomatoes, @lvm, @amazingchow, @anneouyang, @DawnLIU35, @hanke-janson, @ichyycc, @idolpx, @paritosh9199, @gbrugger, @bod17, @NicolasVidalDuque, @ImShalev, @rahuldadz, @CONVORDIV, @PresidentNick, @zhuzhuxia480, @Dec1n, @mmilewczyk, @lladawn, @Sitaras, @Keeplee33, @Kaleb47, @ugeblainebradypp, @krittakondev, @vFrankPZhang, @JustCallMe-AK, @bundubuduri, @AntonioRas, @theedtron, @yjw258, @cs-w0r1d, @joshua2049, @Payne0, @holden-nelson, @jimmy8me7, @oldmonad, @softish, @ivancevich, @AndresNaza, @chuck940, @Jae-eun, @96piku, @tomschr, @HugoDoyon, @Aaron0Dawn, @jarbus, @tonydero, @alfonso21M, @Filipsys, @maxifom, @0xcoder101, @whysokara, @jgh2i, @hanguyen-asana, @azureyang, @0xkzam, @ihrifat2, @nafi-ansary, @jvhvvvn, @cicimen, @cligrenet, @izabelakowal, @zhujun98, @radhikax, @nassersala, @Derek-X-Wang, @TheBatmanIRL, @SoumojitNag, @tarvos21, @abidpu19, @devmarkpro, @likeaixi

**Pull request merged**